pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
 ]
 
 dependencies = [
-  "torch>=2.1.0",
+  "torch>=2.1.0, <=2.5.0",
```

Comment:
does this break with newer torches?

Reply:
It would break, but I believe the commit [1316272](https://github.com/IBM/terratorch/pull/179/commits/13162727f20219185733ec1705fe5bc25c2982ae) will solve that.